nncf/common/graph/patterns/patterns.py

```diff
@@ -339,6 +339,7 @@ class HWFusedPatternNames(Enum):
     LINEAR_ACTIVATIONS_SCALE_SHIFT = PatternDesc("linear_activations_scale_shift")
     LINEAR_ARITHMETIC = PatternDesc("linear_arithmetic")
     LINEAR_ARITHMETIC_ACTIVATIONS = PatternDesc("linear_arithmetic_activations")
+    LINEAR_ARITHMETIC_ACTIVATIONS_ARITHMETIC = PatternDesc("linear_arithmetic_activations_arithmetic")
```

Comment:
Are there any observations about the accuracy & performance improvement/degradations on real cases?

Reply:
Unfortunately, it did not show huge accuracy improvement because runtime removes these FQs before execution. I consider such FQs as redundant and which can lead to accuracy degradation.